neat/tools/fittools/compartmentfitter.py

```diff
@@ -886,17 +986,38 @@ def plotKernels(self, alphas, phimat, t_arr=None):
                 pstring = '%d $\leftrightarrow$ %d'%(ii,jj)
                 ax.set_title(pstring, pad=-10)
 
-    def checkPassive(self, loc_arg, alpha_inds=[0], recompute=False, n_modes=5,
-                           use_all_chans_for_passive=True, force_tau_m_fit=False):
+    def checkPassive(self, loc_arg, alpha_inds=[0], n_modes=5,
+                           use_all_channels_for_passive=True, force_tau_m_fit=False,
+                           recompute=False, pprint=False):
+        """
+        Checks the impedance kernels of the passive model.
+
+        Parameters
+        ----------
+        loc_arg: list of locations or string (see documentation of
+                :func:`MorphTree._convertLocArgToLocs` for details)
+            The compartment locations
+        n_modes: int
+            The number of eigen modes that are shown
+        alpha_inds: list of ints
+            Indices of all mode time-scales to be included in the fit
+        use_all_channels_for_passive: bool
+            Uses all channels in the tree to compute coupling conductances
+        force_tau_m_fit: bool
+            Force using the local membrane time-scale for capacitance fit
+        recompute: bool
+            whether to force recomputing the impedances
+        pprint: bool
+            is verbose if ``True``
+        """
```

Comment:
i would add
```
Returns
-------
None
```
to make it clear that it doesn't return anything rather than "we forgot to document it"